tests/pages/base.py

```diff
@@ -34,6 +34,11 @@ def footer(self):
     def newsletter(self):
         return NewsletterEmbedForm(self)
 
+    def download_button(self, locator):
```

Comment:
I think we should add a docstring for this, or potentially mark is as private. It could be confusing for a test writer as this is only meant to be used from the page objects and not directly from a test.